ibm/resource_ibm_container_worker_pool.go

```diff
@@ -205,7 +204,6 @@ func resourceIBMContainerWorkerPoolRead(d *schema.ResourceData, meta interface{}
 
 	machineType := workerPool.MachineType
 	d.Set("worker_pool_name", workerPool.Name)
-	d.Set("machine_type", machineType)
```

Comment:
Why you removed d.Set()?

Reply:
@hkantare By default the disk encryption is true and machine type returned by the API has appended with `.encrypted` As machine_type is Force_New attribute suggested to remove d.Set rather than checking for ".encrypted"  and "disk_encryption" everytime we set this back.